app/src/ui/notification/new-commits-banner.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react'
+import { Ref } from '../lib/ref'
+import { Octicon, OcticonSymbol } from '../octicons'
+import { Branch } from '../../models/branch'
+
+interface INewCommitsBannerProps {
+  /**
+   * The number of commits behind `baseBranch`
+   */
+  readonly commitsBehindBaseBranch: number
+
+  /**
+   * The target branch that will accept commits
+   * from the current branch
+   */
+  readonly baseBranch: Branch
+}
+
+/**
+ * Banner used to notify user that there branch is _commitsBehind_
```

Comment:
Does `_commitsBehind_` need to be emphasized here? Can we just use human-friendly words here? Something like:

>  Banner used to notify user that there branch is a number of commits behind the base branch